init.cmd

```diff
@@ -0,0 +1,51 @@
+@ECHO OFF

```

Comment:
If it's important to you to leave this script in the root, how do you feel about renaming it to `InitWindowsEnvironment.cmd`? The usage will be the exact same as what you have now: `ini<tab><enter>` but its purpose is clearer that way. Though personally I still think it would be nicest to put it in `Scripts\Windows` and follow the same pattern as all of our scripts.

Reply:
I'm rethinking these scripts as I investigate the macOS counterparts. Just running init.sh doesn't leave the environment variables exported. A user would need to source the script.

We can either be ok with that or just make these scripts pre-reqs to all other scripts and remove the convenience macros. Opinions?

Reply:
Yea I've always thought of the scripts _as_ the convenience macros. I'd be just fine with dropping those, and having each script include the base script to inherit the environment variables.